js/openfisca.js

```diff
@@ -11,55 +10,47 @@ module.exports = {
 }
 
 
-function serialize(form) {
-	var result = [],
-		elements = form.elements
-
-	Array.prototype.forEach.call(elements, function(element) {
-		if (! element.name)
-			return
-
-		var value = element.value
+const serialize = form =>
```

Comment:
If attributing to a variable just for a function declaration, a named `function serialize(form)` still works well.